app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command):
+    name_in, name_out = command.split()[1:]
+
+    if name_in == name_out:
```

Comment:
Make sure the correct command `cp` was entered

Reply:
Ok, done.